pkg/cli/common.go

```diff
@@ -189,6 +216,35 @@ func GetBudFlags(flags *BudResults) pflag.FlagSet {
 	return fs
 }
 
+// GetBudFlagsCompletions returns the FlagCompletions for the common bud flags
+func GetBudFlagsCompletions() commonComp.FlagCompletions {
+	flagCompletion := commonComp.FlagCompletions{}
+	flagCompletion["arch"] = commonComp.AutocompleteNone
+	flagCompletion["annotation"] = commonComp.AutocompleteNone
+	flagCompletion["authfile"] = commonComp.AutocompleteDefault
+	flagCompletion["build-arg"] = commonComp.AutocompleteNone
+	flagCompletion["cache-from"] = commonComp.AutocompleteNone
+	flagCompletion["cert-dir"] = commonComp.AutocompleteDefault
+	flagCompletion["creds"] = commonComp.AutocompleteNone
+	flagCompletion["file"] = commonComp.AutocompleteDefault
+	flagCompletion["format"] = commonComp.AutocompleteNone
+	flagCompletion["iidfile"] = commonComp.AutocompleteDefault
+	flagCompletion["label"] = commonComp.AutocompleteNone
+	flagCompletion["logfile"] = commonComp.AutocompleteDefault
+	flagCompletion["loglevel"] = commonComp.AutocompleteDefault
+	flagCompletion["timestamp"] = commonComp.AutocompleteNone
+	flagCompletion["os"] = commonComp.AutocompleteNone
+	flagCompletion["platform"] = commonComp.AutocompleteNone
+	flagCompletion["runtime-flag"] = commonComp.AutocompleteNone
+	flagCompletion["sign-by"] = commonComp.AutocompleteNone
+	flagCompletion["signature-policy"] = commonComp.AutocompleteNone
+	flagCompletion["tag"] = commonComp.AutocompleteNone
+	flagCompletion["target"] = commonComp.AutocompleteNone
+	flagCompletion["jobs"] = commonComp.AutocompleteNone
+	return flagCompletion
```

Comment:
A bit of a pain, but could you put these in alpha order please?

Reply:
Yes I can. I have them in the same order as they are defined above. I will also sort them.